test/tests_for_sdkmr/ee2_scheduler_test.py

```diff
@@ -16,63 +29,51 @@
 class ExecutionEngine2SchedulerTest(unittest.TestCase):
     @classmethod
     def setUpClass(cls):
-        deploy = os.environ.get("KB_DEPLOYMENT_CONFIG", "test/deploy.cfg")
-        cls.condor = Condor(deploy)
+        cls.condor = Condor(get_ee2_test_config())
         cls.job_id = "1234"
         cls.user = "kbase"
-        cls.catalog_utils = CatalogUtils(
-            url="https://ci.kbase.us/services/Catalog", admin_token="123"
-        )
-
-    @classmethod
-    def tearDownClass(cls):
-        if hasattr(cls, "wsName"):
-            cls.wsClient.delete_workspace({"workspace": cls.wsName})
-            print("Test workspace was deleted")
-
-    def _create_sample_params(self, cgroups):
-        params = dict()
-        params["job_id"] = self.job_id
-        params["user_id"] = "kbase"
-        params["token"] = "test_token"
-        rr = CatalogUtils.normalize_job_settings(cgroups)
-
-        print(rr)
-        params["cg_resources_requirements"] = rr
-
-        return params
 
-    def test_empty_params(self):
```

Comment:
Again, all parameter checking has been moved to the `JobSubmissionParamters` class.